tests/gocase/unit/disk/disk_test.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package disk
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	"github.com/apache/incubator-kvrocks/tests/gocase/util"
+	"github.com/go-redis/redis/v9"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDisk(t *testing.T) {
+	srv := util.StartServer(t, map[string]string{
+		"rocksdb.compression":       "no",
+		"rocksdb.write_buffer_size": "1",
+	})
+	defer srv.Close()
+	ctx := context.Background()
+	rdb := srv.NewClient()
+	defer func() { require.NoError(t, rdb.Close()) }()
+	estimationFactor := 0.5
+	t.Run("Disk usage String", func(t *testing.T) {
+		require.NoError(t, rdb.Set(ctx, "stringkey", "aaaaaaaaaaaaaaaaaaaaaaaa", 0).Err())
+		val, err := rdb.Do(ctx, "Disk", "usage", "stringkey").Int()
+		require.NoError(t, err)
+		require.GreaterOrEqual(t, val, 1)
```

Comment:
IMHO, the expected value is weird. And I wonder if it can be a range rather than only a minimum value.

Reply:
> IMHO, the expected value is weird. And I wonder if it can be a range rather than only a lower bound.

sounds good. Let me try.

Reply:
@PragmaTwice Now my judgment is that the result will not exceed the upper bound or the lower bound within an order of magnitude